rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/ReSubmitJobCommand.java

```diff
@@ -0,0 +1,169 @@
+                                                                     inputStream,
+                                                                     map(this.variables));
+                } else {
+                    jobId = currentContext.getRestClient().submitJobArchive(currentContext.getSessionId(),
+                                                                            inputStream,
+                                                                            map(this.variables));
+                }
+            }
+            writeLine(currentContext, "Job('%s') successfully submitted: job('%d')", pathname, jobId.getId());
+            resultStack(currentContext).push(jobId);
+        } catch (Exception e) {
+            handleError(String.format("An error occurred while attempting to submit job('%s'):", pathname),
+                        e,
+                        currentContext);
+        }
+
+    }
+
+    private void validateFilePath(ApplicationContext currentContext) throws JobCreationRestException {
+
+        if (!isFilePathValid(pathname)) {
+            throw new CLIException(REASON_INVALID_ARGUMENTS, String.format("'%s' is not a valid file.", pathname));
+        }
+
+        if (!isValidFileMimeType(pathname)) {
+            throw new JobCreationRestException("Unknown job descriptor type: " + pathname);
+        }
+
+        if (isFileExisting(pathname) && isFileEmpty(pathname)) {
+            throw new CLIException(REASON_FILE_EMPTY, String.format("'%s' is empty.", pathname));
+        }
+
+        if (!isFileExisting(pathname)) {
+            throw new CLIException(REASON_INVALID_ARGUMENTS, String.format("'%s' does not exist.", pathname));
+        }
+
+    }
+
+    private Map<String, String> map(String variables) {
+        return jobKeyValueTransformer.transformVariablesToMap(variables);
+    }
+
+    private Boolean isFileEmpty(String pathname) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(pathname))) {
+            if (isFileExisting(pathname)) {
+                if (reader.readLine() == null) {
+                    return true;
+                }
+                return false;
+            }
+            return true;
+        } catch (IOException e) {
+            logger.log(Level.INFO, "Error reading file " + pathname, e);
+            return false;
+        }
+    }
+
+    private Boolean isFileExisting(String pathname) {
+        File file = new File(pathname);
+        if (file.exists() && !file.isDirectory())
+            return true;
+        return false;
+    }
+
+    private Boolean isValidFileMimeType(String pathname) {
+        String contentType = URLConnection.getFileNameMap().getContentTypeFor(pathname);
+        if (contentType != null)
+            return (contentType.toLowerCase().equals("application/xml") ||
+                    contentType.toLowerCase().equals("application/zip"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)